src/index.js

```diff
@@ -21,6 +18,7 @@ class Orejs {
     Object.assign(this, eos)
     Object.assign(this, instrument)
     Object.assign(this, ore)
+    Object.assign(this, rightRegistry)
```

Comment:
This should be rightsRegistry (with an s)

Reply:
done